_posts/2019-05-06-lmt-3.2.6.md

```diff
@@ -6,5 +6,5 @@ tags: new-release
 The Lustre Monitoring Tools (LMT) repo is a distributed system that provides a "top"-like display of the activity levels of the server-side (MDS, OSSes, and LNET routers) nodes of one or more Lustre-based filesystems. This release includes support for Lustre 2.12.0.
 
 Learn more:
-- [Release notes](https://github.com/LLNL/lmt/releases/tag/3.2.6)
```

Comment:
Should this actually point to the tag, since the post is about this specific release?